package.json

```diff
@@ -39,6 +39,7 @@
     "minimatch": "3.1.2",
     "minimist": "1.2.6",
     "mocha": "10.2.0",
+    "nightwatch": "^3.4.1",
```

Comment:
Can you remove this?

Reply:
yes i can i will push changes in a minute. thnaks for the guidiance